packages/cx/src/widgets/form/Slider.js

```diff
@@ -308,6 +311,18 @@ class SliderComponent extends VDOM.Component {
       }
    }
 
+   subscribeOnWheel(el) {
+      if (this.unsubscribeOnWheel) {
+         this.unsubscribeOnWheel();
+         this.unsubscribeOnWheel = null;
+      }
+      if (el) {
+         this.unsubscribeOnWheel = addEventListenerWithOptions(el, "wheel", (e) => this.onWheel(e), {
+            passive: false,
+         });
+      }
+   }
```

Comment:
There is one subtle issue here. If a Slider is put inside a Repeater, the same slider widget will be instantiated many times. The rule is that `this` should be used only for things that are reusable across all instances of the widget. In this example, you should use `instance.unsubscribeOnWheel` instead of `this.unsubscribeOnWheel`.

Reply:
Fixed!